src/Php73/Php73.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Symfony\Polyfill\Php73;
+
+/**
+ * @author Gabriel Caruso <carusogabriel34@gmail.com>
+ * @author Ion Bazan <ion.bazan@gmail.com>
+ *
+ * @internal
+ */
+final class Php73
+{
+    const NANO_IN_SEC = 1e9;
+    const NANO_IN_MSEC = 1e3;
+
+    private static $startAt = null;
+    private static $startAtArr = null;
+
+    /**
+     * @param bool $asNum
+     *
+     * @return array|float|int
+     */
+    public static function hrtime($asNum = false)
+    {
+        if ($asNum) {
+            if (is_null(static::$startAt)) {
```

Comment:
use `self::` to access private static properties (here, as it is final, it won't allow breaking stuff by creating a child class, but it still pushes more work at runtime)